### ISSUE TITLE
Add ALT text to all visuals like charts, avatar

### ISSUE THREAD

**Issue by matsbryntse** [bryntum/support#10499]
Chart missing ALT text

https://bryntum.com/products/grid/examples/nested-grid-with-chart/
http://lh/bryntum-suite/schedulerpro/examples/embedded-chart/
